easybuild/easyconfigs/a/advisor/advisor-2020_update2.eb

```
@@ -28,11 +28,12 @@ sanity_check_paths = {
 allow_prepend_abs_path = 'True'
 modextrapaths = {
     'PATH': '%(name)s/bin64',
+    'INTEL_PYTHONPATH': '%(name)s/pythonapi',
     'PYTHONPATH': '%(name)s/pythonapi',
     'INTEL_LICENSE_FILE': '/opt/intel/licenses',
 }
 modextravars = {
-    'ADVISOR_2019_DIR': '%(installdir)s/%(name)s',
+    'ADVISOR_2020_DIR': '%(installdir)s/%(name)s',
```

```suggestion
    'ADVISOR_%(version)s_DIR': '%(installdir)s/%(name)s',
```
Would this work? So that you don't have to change it manually in the future?

---

yes, i'll do it next time